Sources/UInt128.swift

```diff
@@ -603,18 +597,18 @@ extension UInt128 : CustomStringConvertible {
     /// - returns:
     ///     String representation of the stored UInt128 value.
     internal func _valueToString(radix: Int = 10, uppercase: Bool = true) -> String {
-        precondition(radix > 1 && radix < 37, "radix must be within the range of 2-36.")
+        precondition((2...36).contains(radix), "radix must be within the range of 2-36.")
```

Comment:
Any particular reason for changing the logic here?

Reply:
logic is the same, it's just that the range communicates the intent clearer.

Reply:
I guess I can see how it is a bit clearer, so sure, why not